Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -287,7 +287,13 @@ For example if both `READ_EXTERNAL_STORAGE` and `WRITE_EXTERNAL_STORAGE` are lis
 
 ### Permission Analysis
 
-Always check whether the application is requesting permissions it actually needs. Make sure that no permissions are requested which are not related to the goal of the app. For instance: a single-player game that requires access to `android.permission.WRITE_SMS`, might not be a good idea.
+Always check whether the application is requesting permissions it actually requires. Make sure that no permissions are requested which are not related to the goal of the app.
+
+In particular, care should be taken around `DANGEROUS` and `SIGNATURE` permissions as these can affect both the user and the application if mishandled. Take for instance a single-player game that requires access to `android.permission.WRITE_SMS`, this goes against what one would commonly expect from such an application and may indicate potentially malicious usage of the permission.
```

Comment:
```suggestion
Always check whether the application is requesting permissions it actually requires. Make sure that no permissions are requested which are not related to the goal of the app, especially `DANGEROUS` and `SIGNATURE` permissions, since they can affect both the user and the application if mishandled. For instance, it should be suspicious if a single-player game app requires access to `android.permission.WRITE_SMS`.
```